lib/autoproj/daemon/git_api/json_facade.rb

```diff
@@ -29,15 +29,15 @@ def initialize(git_url, model)
                 end
 
                 def ==(other)
-                    @model == other.model
+                    @git_url == other.git_url && @model == other.model
```

Comment:
In practice, outside of tests, probably not since we use this mostly for pull requests and there are URLs in the model (at least on GitHub).

In theory ? Yes. JSONFacade.new(someurl, {}) != JSONFacade.new(otherurl, {}).

Since it is pretty simple ... I thought that making the tests more robust would not hurt ...